src/libpostal_data

```diff
@@ -79,9 +78,13 @@ function download_multipart() {
         else
             max=$size;
         fi;
-        printf "%s\0%s\0%s\0%s\0%s\0" "$i" "$offset" "$max" "$url" "$part_filename"
+        download_part "$i" "$offset" "$max" "$url" "$part_filename" &
+        # wait every time we have started $num_workers processes
+        [ $((i%num_workers)) -eq 0 ] && wait
```

Comment:
Use shell job control and modulo arithmetic to restrict maximum simultaneous downloads, could have also used `xargs -t` to show per download commands, or `xargs external_cmd`, or `xargs bash $STRING_OF_ALL_CMDS` just as easily. This was just closest to my original change.